tasks/4.js

```diff
@@ -0,0 +1,25 @@
+var arr = [1, 2, 1, 3, 4];
+
+function mapArr(arr) {
+  var newArr = [];
+  arr.map(function (x) {
+    if (!newArr.includes(x)) {
+      newArr.push(x);
+    }
+  });
+
+  return newArr;
+}
+
+function filterArr(arr) {
+  var newArr = arr.filter(function (a, b) {
```

Comment:
It would be best to name the variables more correctly as now you need to remember or search, that the second argument of the filter function is an index of the current array item. 🤔

Reply:
That is true. Thanks.